src/NUnitFramework/framework/Internal/AssemblyHelper.cs

```diff
@@ -159,7 +159,7 @@ public static Assembly Load(string nameOrPath)
 
         private static bool IsFileUri(string uri)
         {
-            return uri.ToLower().StartsWith(UriSchemeFile);
+            return uri.StartsWith(UriSchemeFile, StringComparison.CurrentCultureIgnoreCase);
```

Comment:
I would like to change behavior here to OrdinalIgnoreCase if we can without breaking anyone. CurrentCulture and InvariantCulture are slower and semantically correct for linguistic operations, not for canonical identifiers.

Reply:
I think this should be a fairly safe change. I nearly did this myself before submitting the initial PR, but I opted for not doing it mostly out of abundance of caution.